### PR TITLE
add configs of hrhrnet and hrnet

### PR DESCRIPTION
**Pull request by zhiboniu**
add configs of hrhrnet and hrnet

### CODE REVIEW

File: configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+use_gpu: true
```
Comment (qingqing01) [inline configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml]: configs/keypoint/higherhrnet/xx

File: configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+flip_perm: &flip_perm [0, 2, 1, 4, 3, 6, 5, 8, 7, 10, 9, 12, 11, 14, 13, 16, 15]
+input_size: &input_size 512
+hm_size: &hm_size 128
+hm_size_2x: &hm_size_2x 256
+max_people: &max_people 30
+metric: COCO
+IouType: keypoints
+num_classes: 1
+
+LearningRate:
```
Comment (qingqing01) [inline configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml]: 顺序的话，建议:

全局参数
Arch
LR + Opt
最后Dataset

File: configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml (hunk 1 above)
Comment (zhiboniu) [reply configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml]: done

File: configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml (hunk 2 above)
Comment (zhiboniu) [reply configs/keypoint_higherhrnet/higherhrnet_hrnet_v1_512.yml]: done

**Review by qingqing01 — DISMISSED** [review]

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**